packages/volto-slate/src/widgets/RichTextWidgetView.jsx

```diff
@@ -20,16 +19,16 @@ export const SlateRichTextWidgetView = ({
   intl,
 }) => {
   const Block = config.blocks.blocksConfig.slate.view;
-  return value ? (
-    <ErrorBoundary
-      name={intl.formatMessage(messages.error, { name: className })}
-    >
-      <div className={cx(className, 'slate', 'widget')}>
-        <Block data={{ value: value }}>{children}</Block>
-      </div>
-    </ErrorBoundary>
-  ) : (
-    ''
+  return (
+    value ?? (
```

Comment:
I think this is wrong. If there is a `value`, it will be displayed directly, without going through error handling. I think the way it was before is correct.